Lecture-6/Task4/l6task4/Application.java

```diff
@@ -0,0 +1,91 @@
+package l6task4;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.LinkedList;
+import java.util.List;
+
+public class Application {
+
+    private static SimpleDateFormat formatForDate = new SimpleDateFormat("hh:mm:ss:SSS");
+
+    private static List<Double> getArrayList() {
+        List<Double> arrayList = new ArrayList<>();
+        int counter = 0;
+        while (counter < 10000000) {
+            arrayList.add(Math.random());
+            counter++;
+        }
+        return arrayList;
+    }
+
+    private static List<Double> getLinkedList() {
```

Comment:
This method duplicates `getArrayList()`, refactor them both and reuse

Reply:
done